torch/nested/_internal/ops.py

```diff
@@ -151,6 +151,7 @@ def inner(*args, **kwargs):
 
             for table in tables:
                 table[aten_op] = get_inner(aten_op)
+        return func
```

Comment:
how did this work before :0

Reply:
I think you mentioned we never tried running `is_contiguous` with the memory format overload, and was the only case where we referenced a function that was decorated lol